tests/store/api/conftest.py

```diff
@@ -536,42 +536,6 @@ def fixture_store_with_pools_for_multiple_customers(
     yield store
 
 
-@pytest.fixture(name="store_with_analyses_for_cases")
```

Comment:
I think some of my tests depend on this fixture

Reply:
I could not find any. Even so, putting the fixture one level up still keeps it available for any test in the directories below